addons/zeus/stringtable.xml

```diff
@@ -1112,6 +1112,10 @@
             <English>Random filling</English>
             <French>Remplir au hasard</French>
         </Key>
+        <Key ID="STR_ACE_Zeus_ModuleGarrison_TeleportText">
```

Comment:
https://github.com/acemod/ACE3/blob/98da86b74d4089bc3403d0b0c0074e6a46ce35d8/addons/respawn/stringtable.xml#L29-L38

Reply:
What do you suggest? 
That I take the first word of the string? It could be different on asian languages.

Reply:
Just that we have some of them, I have no clue which are correct without the "base" part as well, might as well just leave it.